### PR TITLE
Added dSPM

### PR DESCRIPTION
**Pull request by klankinen**
Added 'dSPM' to the glossary.

### CODE REVIEW

File: doc/glossary.rst [hunk 1]
```diff
@@ -54,6 +54,12 @@ general neuroimaging concepts. If you think a term is missing, please consider
     dipole
         See :term:`equivalent current dipole`.
 
+    dSPM
+        Dynamic statistical parametric mapping (abbr. ``dSPM``) gives a noise-
+        normalized :term:`source estimate` at a given source location. dSPM 
+        can be obtained by applying the inverse operator to the signal 
+        covariance matrix.
```
Comment (larsoner) [inline doc/glossary.rst]: This is a little bit vague. Do you mean the "dSPM normalization terms can be obtained ..." ?

**Comment by drammock** [comment]
There is [an error building the docs](https://circleci.com/gh/mne-tools/mne-python/16016?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) with CircleCI (our documentation test server). It says

```
<rst_prolog>:56: WARNING: term not in glossary: source estimate
```

The reason is that the title of the relevant entry is

```
source estimates (abbr. ``stc``)
```

...which is hard to link to.  Can you change the title of the source estimates entry to look more like the entry for "projector" (i.e., where the abbreviation is not part of the title?)  And then also make sure the cross-reference in your dSPM entry matches it (i.e., singular vs plural)

File: doc/glossary.rst [hunk 2]
```diff
@@ -54,6 +54,12 @@ general neuroimaging concepts. If you think a term is missing, please consider
     dipole
         See :term:`equivalent current dipole`.
 
+    dSPM
+        Dynamic statistical parametric mapping (abbr. ``dSPM``) gives a noise-
+        normalized :term:`source estimate` at a given source location. dSPM is
```
Comment (agramfort) [inline doc/glossary.rst]:
```suggestion
        normalized :term:`minimum norm estimate` at a given source location. dSPM is 
```

File: doc/glossary.rst [hunk 3]
```diff
@@ -54,6 +54,12 @@ general neuroimaging concepts. If you think a term is missing, please consider
     dipole
         See :term:`equivalent current dipole`.
 
+    dSPM
+        Dynamic statistical parametric mapping (abbr. ``dSPM``) gives a noise-
+        normalized :term:`source estimate` at a given source location. dSPM is 
+        calculated by dividing the activity estimate at each source location by 
+        the noise sensitivity estimate.
```
Comment (agramfort) [inline doc/glossary.rst]:
```suggestion
        the baseline standard deviation of the noise.
```

**Comment by larsoner** [comment]
Thanks @klankinen

**Pull request merged**